docs/tests/screenshotter.md

```diff
@@ -0,0 +1,40 @@
+# Screenshotter
+
+ScreenShotter automates visual regression testing of your responsive web UI by comparing DOM screenshots over time.
+
+This is an internal tool that's being used in Elementor's CI process, in order to make sure that the PR changes don't
+break the original behavior of the product.
```

Comment:
```suggestion
This is an internal tool that's being used in Elementor's CI process, in order to make sure that the PR changes don't break the original behavior of the product.
```

Reply:
That's intentional, in order to keep a reasonable line length:
![image](https://user-images.githubusercontent.com/32631382/167283411-f2a8cc4d-af87-44cc-b88a-b7555998f34e.png)